frontend/src/pages/patients/PatientsPage.tsx

```diff
@@ -65,9 +76,144 @@ function patientAliasFilterWhereVariables(value: string): PatientAliasWhere[] {
   }
 }
 
-export const PatientsPage: React.FunctionComponent = (props) => {
+function addCDashToCMOId(cmoId: string): string {
+  return cmoId.length === 6 ? `C-${cmoId}` : cmoId;
+}
+
+const phiWarning = {
```

Comment:
no biggie but it's a good practice to make "content" constants like this all caps.  e.g. PHI_WARNING.  This distinguishes them from variables that are more logic related